packages/spindle-ui/src/SegmentedControl/SegmentedControl.stories.example.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { Button } from '../Button';
+import { SegmentedControl } from './SegmentedControl';
+
+export function SegmentedControlUpdateSelectedIdExample() {
+  const [bool, setBool] = useState(false);
+  const handleButtonCLick = () => setBool((prevState) => !prevState);
+  return (
+    <div>
+      <Button size="medium" onClick={handleButtonCLick} type="button">
+        渡しているselectedIdを更新
```

Comment:
nits of nits
なんのstoriesなのかは前後の文章で補足している & やや冗長なので、
```suggestion
        selectedIdを更新
```
等で良さそうに思いました 👌🏻